### ISSUE TITLE
change or disable shortcut?

### ISSUE THREAD

**Issue by hashcatX** [benbuck/rbtray#114]
Hi.

minimzing an application is done via shortcut ctrl+alt+arrowdown.
Can i change or disable that? It clashes with other apps.

thxs^^

**Comment by Chealer** [comment]
No, it cannot be done. Please do not ask questions here, this is an ITS.

**Comment by benbuck** [comment]
Hi @hashcatX,

Thanks for the suggestion. I've decided not to add any more features to RBTray, so I'm marking this as "Won't Fix". However, I've just today created a pre-release of a program called [Finestray](https://github.com/benbuck/finestray), which can also minimize windows to the tray, and should already support this feature. Feel free to give it a try to see if it could help for your needs.

Thanks,
Benbuck
